rules/opentitan_test.bzl

```diff
@@ -420,6 +420,7 @@ def opentitan_functest(
                 },
                 "cw340": {
                     "cw340": "@//hw/bitstream/cw340",
+                    "hyper340": "@//hw/bitstream/cw340",
```

Comment:
I suppose the ribbon cable patch panel of CW340 enables us to use the exact same bitstream for both transports, unlike CW310?

Reply:
Yes, for now we only have one bitstream.